xsd/netex_framework/netex_responsibility/netex_entity_support.xsd

```diff
@@ -118,19 +118,23 @@ Rail transport, Roads and Road transport
 			<xsd:documentation>Reference to a Type of an ENTITY.</xsd:documentation>
 		</xsd:annotation>
 	</xsd:element>
-	<xsd:complexType name="ClassRefStructure">
+	<xsd:complexType name="ClassRefStructure" mixed="true">
 		<xsd:annotation>
 			<xsd:documentation>Type for a reference to the Class of a ENTITY.</xsd:documentation>
 		</xsd:annotation>
-		<xsd:simpleContent>
-			<xsd:extension base="xsd:string">
-				<xsd:attribute name="nameOfClass" type="NameOfClass" use="required">
-					<xsd:annotation>
-						<xsd:documentation>Name of referenced Class.</xsd:documentation>
-					</xsd:annotation>
-				</xsd:attribute>
-			</xsd:extension>
-		</xsd:simpleContent>
+		<xsd:sequence>
+			<xsd:element name="MemberRef" type="ClassRefStructure" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Why a recursive definition here (we are within the ClassRefStructure definition) ?
And the result is a bit weird
![image](https://github.com/NeTEx-CEN/NeTEx/assets/3328999/392cdd8d-e27c-4da1-ab63-03fe5e2d6bb0)

Reply:
@Aurige  Can't a member have member again? I think recursive is correc.t Or what would be your proposal?

Reply:
input from Christophe needed

Reply:
That's a ClassRefStructure  inside a ClassRefStructure referred as Member ... but I'm not sure about what we wan't to say here, and waht is a member classs of a class ? Do you mean nested (for example a Quay inside a StopPlace) ? and if so, how would you make the difference between a ref (i.e a QuayRef) and the full object inclusion (Quay) ?

Reply:
The original task is from Stefan. So I ask hiim:

In order to make a profile definition TypeOfFrame/ClassInFrameRef is an embedded way to describe a profile. What is currently lacking is the ability to describe the elements required in the profile.

If from this minimal definition a derived XML Schema (or Schematron) can be generated, every profile can be validated against an always up to date NeTEx upstream, without manual the manual labor of maintaining an extra XML Schema by hand (such as EPIP, Dutch, etc.)

@skinkie  how deep do we want to go and Ref and objects?